core/src/test/scala/integration/kafka/api/PlaintextConsumerTest.scala

```diff
@@ -189,6 +189,27 @@ class PlaintextConsumerTest extends BaseConsumerTest {
     assertThrows(classOf[InvalidTopicException], () => consumer.partitionsFor(";3# ads,{234"))
   }
 
+  @ParameterizedTest(name = TestInfoUtils.TestWithParameterizedQuorumAndGroupProtocolNames)
+  @MethodSource(Array("getTestQuorumAndGroupProtocolParametersAll"))
+  def testSubscribeInvalidTopic(quorum: String, groupProtocol: String): Unit = {
```

Comment:
great addition, but this test should go in the `PlainTextConsumerSubscriptionTest` file.